sentry-ruby/lib/sentry/interfaces/single_exception.rb

```diff
@@ -15,7 +15,15 @@ class SingleExceptionInterface < Interface
 
     def initialize(exception:, stacktrace: nil)
       @type = exception.class.to_s
-      @value = (exception.message || "").byteslice(0..Event::MAX_MESSAGE_SIZE_IN_BYTES)
+      exception_message =
+        if exception.respond_to?(:detailed_message)
+          exception.detailed_message(highlight: false)
```

Comment:
- `highlight: false` removes ANSI escape code that's mainly for standard output.
- `#detailed_message` always returns a string. Even if the exception's message is `nil`, it still returns `"(Exception)"`.